src/css/Book.css

```diff
@@ -1,46 +1,98 @@
 .BookCard {
-  width: 75rem;
-  height: 2.625rem;
-  margin: 2.313rem 6.25rem 0;
+  margin: 0 0 2.313rem;
   padding: 2rem 9.188rem 1.625rem 1.688rem;
   border-radius: 4px;
-  border: 4px solid orange;
-  background-color: #cec0c0;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+}
+
+.BookCard:hover {
+  border: 1px solid #0290ff;
+}
+
+.one {
+  margin-bottom: 0.75rem;
 }
 
-.BookCardContainer {
+.two {
   display: flex;

```

Comment:
- [OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code to No repeating of the same rule by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```css
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- JS file
```js
<div className="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

Reply:
@BenMukebo Thank you very much for giving me this knowledge, I will try to understand this code and try to implement it.
Thank you very much once again for the suggesions.